debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/MMObjectAccessBarrier_V1.java

```diff
@@ -49,9 +50,17 @@ protected MMObjectAccessBarrier_V1() throws CorruptDataException
 		if(J9BuildFlags.gc_compressedPointers) {
 			try {
 				J9JavaVMPointer vm = J9RASHelper.getVM(DataType.getJ9RASPointer());
-				shift = vm.compressedPointersShift().intValue();
+
+				/* use reflection to access compressedPointersShift which will not exist if build is default */
+				UDATA shiftUdata = (UDATA)J9JavaVMPointer.class.getMethod("compressedPointersShift").invoke(vm); //$NON-NLS-1$
+				shift = shiftUdata.intValue();
 			} catch (CorruptDataException cde) {
 				raiseCorruptDataEvent("Error initializing the object access barrier", cde, true);
+			} catch (NoSuchMethodException | SecurityException | IllegalAccessException | InvocationTargetException re) {
+				/* error caused by reflection */
+				CorruptDataException rcde = new CorruptDataException(re.toString());
```

Comment:
Isn't there a constructor that takes the `re` as parameter?

Reply:
yes. just pushed a fix for that